frontend/amundsen_application/static/js/pages/SearchPage/ResourceSelector/index.tsx

```diff
@@ -72,7 +72,7 @@ export class ResourceSelector extends React.Component<ResourceSelectorProps> {
           }
           onChange={this.onChange}
         />
-        <span className="subtitle-2">{option.label}</span>
+        <span>{option.label}</span>
```

Comment:
Let me insist on doing this:
```suggestion
        <span className="text-subtitle-w2">{option.label}</span>
```
Even if the font-weights don't change that much with the browser rendering, it is better to keep it in sync with our designs.